src/renate/utils/misc.py

```diff
@@ -37,3 +39,45 @@ def maybe_populate_mask_and_ignore_logits(
         logits.index_fill_(1, class_mask.to(logits.device), -float("inf"))
 
     return logits, class_mask
+
+
+class AdditionalTrainingMetrics(Callback):
+    def __init__(self) -> None:
+        self._training_times = []
+
+    def on_train_start(self) -> None:
+        self._training_times.append(time.time())
+
+    def on_train_epoch_end(self) -> None:
+        self._training_times.append(time.time())
```

Comment:
instead of using a list, maybe simply save start and end time?

Reply:
I just wanted to keep the flexibility of retrieving all times, though the current interface doesn't
necessarily allow for it.

Reply:
unless we use it or plan to use it soon, we shouldn't sacrifice simplicity for flexibility. unless you strongly disagree, let's keep this simple.